demo/talkingdata-adtracking-fraud-detection/README.md

```diff
@@ -47,15 +47,15 @@ in `train_and_serve.py` to produce new sample csv for training.
 ### Start OpenMLDB cluster
 
 ```
-./init.sh
+/work/init.sh
 ```
 
 ### Start predict server
 
 You can start predict server even if you haven't deployed, with option `--no-init`.
 
 ```
-python3 predict_server.py --no-init > predict.log 2>&1 &
+/work/talkingdata/predict_server.py --no-init > predict.log 2>&1 &
```

Comment:
my mistake, should be 
python3 /work/talkingdata/predict_server.py --no-init > predict.log 2>&1 &